omod/src/main/resources/apps/registrationapp_app.json

```diff
@@ -87,16 +87,22 @@
         {
           "id": "myPersonAttributeLabel",
           "label": "My new zone",
-          "fields": [
+          "questions": [
```

Comment:
all fields need to be in the following hierarchy:
section -> questions -> fields or there will be an error